platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/http/HttpRequestImpl.java

```diff
@@ -0,0 +1,196 @@
+package com.mapbox.mapboxsdk.http;
```

Comment:
Could we move this file to a `modules` folder?

Reply:
Possible yes, but to completely exclude okhttp, the user would also have to exclude `com.mapbox.mapboxsdk.http.HttpRequestUtil` which we can't move due to semver. I'm suggesting postponing changing package names until next major release.